src/projections_site/templates/crs.html.j2

```diff
@@ -4,106 +4,113 @@
 {% block header %}{{ crs | string }} - <i>{{ crs.name }}</i>{% endblock %}
 
 {% block content %}
-    <h2>Attributes</h2>
 
-    <h3>Unit</h3>
-    <p>{{ crs.axis_info[0].unit_name }}</p>
-
-    <h3>Scope</h3>
-    <p>{{ crs.scope }}</p>
+    <div class="row">
+        <h2 class="mt-4 fs-3">Attribute</h2>
+        <hr>
+    </div>
 
-    <h3>Remarks</h3>
-    <p>{{ crs.remarks }}</p>
+    <div class="col-md-4">
+        <h3>Unit</h3>
+        <p>{{ crs.axis_info[0].unit_name }}</p>
 
-    <h3>Area of use</h3>
-    <p>{{ crs.area_of_use.name }}</p>
-    <p>Data Source: EPSG  {{ util.get_epsg_version_string() }}</p>
-    <p>Datum: {{util.get_datum(crs)}}</p>
-    <p>Coordinate System: {{ util.get_coordinate_system(crs) }}</p>
-    <p>Projection: {{ util.get_coordinate_operation(crs) }}</p>
-    <p>Geodetic CRS: {{ util.get_geodetic_crs(crs) }}</p>
+        <h3>Scope</h3>
+        <p>{{ crs.scope }}</p>
 
-    <h2>Export</h2>
-    <ul class="nav nav-tabs">
-        <li class="nav-item">
-            <button
-                class="nav-link active"
-                id="wkt2-tab"
-                data-bs-toggle="tab"
-                data-bs-target="#wkt2-tab-pane"
-                type="button"
-                role="tab"
-                aria-controls="home-tab-pane"
-                aria-selected="true">
-                WKT2
-            </button>
-            <!--<a class="nav-link active" aria-current="page" href="#">WKT2</a>-->
-        </li>
+        <h3>Remarks</h3>
+        <p>{{ crs.remarks }}</p>
+    </div>
 
-        <li class="nav-item">
-          <button
-                class="nav-link"
-                id="json-tab"
-                data-bs-toggle="tab"
-                data-bs-target="#json-tab-pane"
-                type="button"
-                role="tab"
-                aria-controls="json-tab-pane"
-                aria-selected="false">
-              JSON
-          </button>
-        </li>
+    <div class="col-md-4">
+        <h3>Area of use</h3>
+        <p>{{ crs.area_of_use.name }}</p>
```

Comment:
The "Area of use" header here makes it look like everything in that column is related to "area of use", but really it's just this one attribute. Maybe "Area of use" could go in the left column, or we could change to a consistent style of labeling attributes, e.g. always `h3` or always `bold`, or something else :) I don't think this is important for this PR though, just thinking out loud.